qiskit_experiments/library/characterization/analysis/stark_ramsey_fast_analysis.py

```diff
@@ -0,0 +1,303 @@
+        one may apply Fourier transform to the data to estimate the coefficient,
+        but this is not the case.
+
+        This analysis assumes the following polynominal for the phase imparted by the Stark shift.
+
+        .. math::
+
+            \theta_{\text Stark}(x) = 2 \pi t_S f_S(x),
+
+        where
+
+        .. math::
+
+            f_S(x) = c_1 x + c_2 x^2 + c_3 x^3 + f_\epsilon,
+
+        denotes the Stark shift. In the perturbation picture,
+        the Stark shift is a quadratic function of :math:`x`, but linear and cubic term,
+        and also offset are also empirically considered to account for the
+        unwanted effect, e.g. strong drive, collisions, TLS, and so forth,
+        and frequency mis-calibration, respectively.
+
+    # section: fit_model
+
+        .. math::
+
+            F_{X+} = \text{amp} \cdot \cos \left( dt f_S^+(x) \right) + \text{offset}, \\
+            F_{Y+} = \text{amp} \cdot \sin \left( dt f_S^+(x) \right) + \text{offset}, \\
+            F_{X-} = \text{amp} \cdot \cos \left( dt f_S^-(x) \right) + \text{offset}, \\
+            F_{Y-} = \text{amp} \cdot \sin \left( dt f_S^-(x) \right) + \text{offset},
+
+        where
+
+        .. math ::
+
+            f_S^\nu(x) = c_1^\nu x + c_2^\nu x^2 + c_3^\nu x^3 + f_\epsilon.
+
+        The Stark shift is asymmetric with respect to :math:`x=0`, because of the
+        anti-crossings of higher energy levels. In a typical transmon qubit,
+        these levels appear only in :math:`f_S < 0` because of the negative anharmonicity.
+        To precisely fit the results, this analysis uses different model parameters
+        for positive (:math:`x > 0`) and negative (:math:`x < 0`) shift domains.
+
+        To obtain the initial guess, the following calculation is employed in this analysis.
+        First, oscillations in each quadrature are normalized and differentiated.
+
+        .. math ::
+
+            \dot{F}_X = \frac{\partial}{\partial x} \bar{F}_X = dt \frac{d}{dx} f_S \bar{F}_Y, \\
+            \dot{F}_Y = \frac{\partial}{\partial x} \bar{F}_Y = - dt \frac{d}{dx} f_S \bar{F}_X. \\
+
+        The root sum square of above quantities yields
+
+        .. math ::
+
+            \sqrt{\dot{F}_X + \dot{F}_Y} = dt \frac{d}{dx} f_S = dt (c_1 + 2 c_2 x + 3 c_3 x^2).
+
+        By using this synthesized data, one can estimate the initial guess of the
+        polynomial coefficients by quadratic regression.
+        This fit protocol is independently conducted for the experiment data on the
+        positive and negative shift domain.
+
+    # section: fit_parameters
+
+        defpar \rm amp:
+            desc: Amplitude of both series.
+            init_guess: Median of root sum square of Ramsey X and Y oscillation.
+            bounds: [0, 1]
+
+        defpar \rm offset:
+            desc: Base line of all series.
+            init_guess: Roughly the average of the data.
+            bounds: [-1, 1]
+
+        defpar dt:
```

Comment:
I agree. Replaced with `r` in [2a9dc3e](https://github.com/Qiskit-Extensions/qiskit-experiments/pull/1009/commits/2a9dc3ec1d0169f1e594a0aa964d1707d4e78129). Welcome better suggestions.

Reply:
You could just use `t_S` and carry the 2 pi around. Otherwise, I don't know that any symbol is better than another. Maybe something w or tau to hint at width/time.

Reply:
I was thinking 2 pi t_S was bit lengthy but doesn't look really bad. Replaced in [f2b5a39](https://github.com/Qiskit-Extensions/qiskit-experiments/pull/1009/commits/f2b5a3939e8c4542c67da80a42b3165218ab18b3)